src/lib/mina_wire_types/mina_base/mina_base_account_update.ml

```diff
@@ -2,6 +2,7 @@ module Authorization_kind = struct
   module V1 = struct
     (* field for Proof is a verification key hash *)
     type t = Signature | Proof of Snark_params.Tick.Field.t | None_given
+    [@@deriving yojson]
```

Comment:
Please don't add this here; the `wire_types` library should only contain types

Reply:
i've reimplemented this solution to not do this